client/landing/gutenboarding/onboarding-block/fse-beta-opt-in/index.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+	Title,
+	SubTitle,
+	ActionButtons,
+	NextButton,
+	BackButton,
+	SkipButton,
+} from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { Icon, siteLogo, header, navigation, arrowRight } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import useStepNavigation from '../../hooks/use-step-navigation';
+import { useTrackStep } from '../../hooks/use-track-step';
+import { trackEventWithFlow } from '../../lib/analytics';
+import { STORE_KEY as ONBOARD_STORE } from '../../stores/onboard';
+
+import './style.scss';
+
+const FseBetaOptIn: React.FunctionComponent = () => {
+	const { __ } = useI18n();
+	const { goNext, goBack } = useStepNavigation();
+	const { enrollInFseBeta } = useDispatch( ONBOARD_STORE );
+	const { shouldEnrollInFseBeta } = useSelect( ( select ) => select( ONBOARD_STORE ) );
+	const pickBeta = ( shouldEnroll: boolean ) => {
+		enrollInFseBeta( shouldEnroll );
+		goNext();
+	};
+	const optIn = () => {
+		trackEventWithFlow( 'calypso_fse_beta_opt_in' );
+		pickBeta( true );
+	};
+
+	useTrackStep( 'FseBetaOptIn', () => ( {
+		selected_fse_beta_opt_in: shouldEnrollInFseBeta(),
+	} ) );
+
+	return (
+		<div className="gutenboarding-page fse-beta-opt-in">
+			<div className="fse-beta-opt-in__header">
+				<div>
+					<Title>{ __( 'Join Our Full Site Editing Beta' ) }</Title>
+					<SubTitle>{ __( 'Experience the new WordPress.com site editing features' ) }</SubTitle>
+				</div>
+				<ActionButtons className="fse-beta-opt-in__actions">
+					<BackButton onClick={ goBack } />
+					<SkipButton onClick={ () => pickBeta( false ) } />
+				</ActionButtons>
+			</div>
+			<ol className="fse-beta-opt-in__content">
+				<li className="fse-beta-opt-in__item">
+					<Icon icon={ siteLogo } />
+					<h3>{ __( 'Edit Your Logo' ) }</h3>
+					<p>
+						{ __(
+							'Change and update your logo and other header elements without leaving the editor'
```

Comment:
Should we involve Editorial for a review of the copy here?

Reply:
Ya that's a good idea. I'd intended to, thanks for the reminder